DS13/code/modules/overmap/transporter.dm

```diff
@@ -171,18 +238,15 @@
 	else
 		to_chat(user, "This is already in use!")
 
+/obj/machinery/computer/camera_advanced/transporter_control/attack_ai(mob/user)
+	return attack_hand(user)
+
 /obj/machinery/computer/camera_advanced/transporter_control/attack_hand(mob/user)
-	if(!ishuman(user))
-		return
-	var/mob/living/carbon/human/L = user
-	var/obj/item/card/id/ID = L.get_idcard(FALSE)
-	if(ID && istype(ID))
-		if(!check_access(ID))
-			to_chat(L, "You require a security level access card to use this console!.")
-			return FALSE
-	else
-		to_chat(L, "You require a security level access card to use this console!.")
+	if(!isliving(user))
 		return
+	if(!allowed(user))
+		to_chat(user, "You require a security level access card to use this console!.")
```

Comment:
Scotty operates the transporter multiple times, and he's from engineering not security.